cirq/ops/pauli.py

```diff
@@ -93,14 +75,59 @@ def __sub__(self, other_or_shift: Union['Pauli', int]
             return self.difference(other_or_shift)
     # pylint: enable=function-redefined
 
-    def __str__(self):
-        return self._name
 
-    def __repr__(self):
-        return 'cirq.Pauli.{!s}'.format(self)
+class _PauliX(Pauli, common_gates.XPowGate):
+    def __init__(self, *, exponent: Union[value.Symbol, float] = 1.0):
+        super(_PauliX, self).__init__(_index=0, _name='X', exponent=exponent)
+
+    def _value_equality_values_cls_(self):
```

Comment:
This is very strongly coupled to an internal implementation detail of `value_equality`. I would instead re-annotate XPowGate with value equality, to override the options specified by eigen gate.